apps/admin-gui/src/app/shared/components/dialogs/copy-sponsored-members-dialog/copy-sponsored-members-dialog.component.html

```diff
@@ -0,0 +1,71 @@
+<div class="{{theme}}">
+  <h1 mat-dialog-title>{{'DIALOGS.COPY_SPONSORED_MEMBERS.TITLE' | translate}}</h1>
+  <div mat-dialog-content>
+    <ng-template
+      matStepLabel
+      >{{'DIALOGS.COPY_SPONSORED_MEMBERS.SPONSOR_lABEL' | translate}}</ng-template
+    >
+    <h6>{{'DIALOGS.COPY_SPONSORED_MEMBERS.SPONSOR_FROM' | translate}}</h6>
+    <perun-web-apps-user-search-select
+      (userSelected)="sponsorFromChanged($event)"
+      [disableAutoSelect]="true"
+      [users]="data.voSponsors">
+    </perun-web-apps-user-search-select>
+    <ng-template #spinner>
+      <perun-web-apps-loading-table></perun-web-apps-loading-table>
+    </ng-template>
+    <div class="position-relative" *ngIf="firstSearchDone">
+      <h6>{{'DIALOGS.COPY_SPONSORED_MEMBERS.SPONSORED_MEMBERS' | translate}}</h6>
+      <perun-web-apps-alert alert_type="info">
+        {{'DIALOGS.COPY_SPONSORED_MEMBERS.SPONSORED_MEMBERS_INFO' | translate}}
+      </perun-web-apps-alert>
+      <perun-web-apps-immediate-filter
+        (filter)="filter = $event"
+        [placeholder]="'VO_DETAIL.SETTINGS.SPONSORED_MEMBERS.FILTER'">
+      </perun-web-apps-immediate-filter>
+      <app-sponsored-members-list
+        *perunWebAppsLoader="loading; indicator: spinner"
+        [disableRouting]="true"
+        [hidden]="loading"
+        [sponsoredMembers]="filteredSponsoredMembers"
+        [selection]="selection"
+        [filterValue]="filter"
+        [displayedColumns]="['select', 'id', 'name', 'email', 'logins', 'sponsors']"
+        [tableId]="tableId">
+      </app-sponsored-members-list>
+    </div>
+    <app-choose-sponsor
+      *ngIf="!loading"
+      [voId]="data.voId"
+      [copy]="true"
+      [disableSelf]="disableSelf"
+      [voSponsors]="voSponsors"
+      (sponsorTypeSelected)="sponsorType = $event"
+      (sponsorSelected)="sponsorTo = $event">
+    </app-choose-sponsor>
+    <div>
+      <br />
```

Comment:
Just a nitpick, but I think that it looks better without this empty space - but it is just my opinion, so you don't have to change it.
```suggestion
```